packages/stark-ui/src/modules/table/components/column.component.html

```diff
@@ -39,4 +39,9 @@
 			*ngTemplateOutlet="columnTemplate; context: { $implicit: { rowData: rowItem, displayedValue: getDisplayedValue(rowItem) } }"
 		></ng-container>
 	</td>
+	<ng-container *ngIf="footerValue !== 'undefined' && footerValue !== ''">
```

Comment:
In this specific case, we should check only for `undefined`. Because we might set the `footerValue` to `''` if the footer should be enabled because the developer set the `footerValue` for other column..

Reply:
Ok ok I see